helm/cas-ciip-portal/templates/mailhog-route.yaml

```
@@ -0,0 +1,26 @@
+{{- if .Values.mailhog.enable }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{ template "cas-ciip-portal.fullname" . }}-mailhog
+  labels:
+{{ include "cas-ciip-portal.labels" . | indent 4 }}
+
+spec:
+  host: {{ .Values.mailhog.host }}
+  port:
+    targetPort: {{ .Values.service.port.http }}
```

```suggestion
    targetPort: {{ .Values.service.port.http }}
  tls:
    insecureEdgeTerminationPolicy: Redirect
    termination: edge
```
This will give us https